pkg/db/db_local/create_client.go

```diff
@@ -155,6 +157,7 @@ func createMaintenanceClient(ctx context.Context, port int) (*pgx.Conn, error) {
 	// We want to wait for a LONG time for this to complete
 	// Use the context that was given - since that is tied to os.Signal
 	// and can be interrupted
+	statushooks.SetStatus(ctx, "Waiting for recovery")
```

Comment:
is this still needed? when we also have `"Database is recovering. This may take some time."`